stargate-core/app/com/salesforce/mce/stargate/controllers/McSsoController.scala

```diff
@@ -52,8 +52,9 @@ class McSsoController @Inject() (
       id <- request.session.data.get("id")
     } yield (mid.toLong, userId.toLong, id)
 
-    data.fold(Future.successful(Ok.withNewSession)) { case (mid, userId, id) =>
-      sessionTrackingService.destroy(mid, userId, id).map(_ => Ok.withNewSession)
+    val result = Ok.withCookies(Cookie(name = "PLAY_SESSION", value = "", secure = true, sameSite = Some(Cookie.SameSite.None)))
```

Comment:
Sorry, I still don't get where the bug is from. Why do we need to set `SameSite` as `None` or why `Ok.withNewSession` does not do what we want?

Reply:
1. With latest chrome changes browser will not accept cookie unless SameSite attribute is set is set. So the PLAYSESSIOn is not cleared during logout. 
2. Ok.withNewSession does not send SameSite attribute , where as  Ok.withCookie allows this and can be used to clear cookie instead

Reply:
So, Does it mean this is a bug in playframework? and what we do is having a workaround?

Reply:
We should also not hard code "PLAY_SESSION" here neither but getting it from config "play.http.session.cookieName".

Reply:
https://github.com/playframework/playframework/blob/480e814204bf7e1ae5dabce1178da270080f9eba/core/play/src/main/resources/reference.conf#L135-L151 seems most of the things can be achieved by having the right config

Reply:
https://www.playframework.com/documentation/2.8.x/SettingsSession is a better reference for those settings. Thanks to @Shwetank87

Reply:
Stargate uses an older version of scala (2.12.4) that does not support SameSite.None. so this config will not have any effect

Reply:
Scala? You mean Playframework?

Reply:
I've a question and a suggestion.

Can we do `Ok.withNewSession.withCookies(Cookie(sameSite = Option(Cookie.SameSite.None)))` ? instead. From Reading the code and play documentation it appears that it should do what we want, and we wouldn't need to manually clear out the value of PLAY_SESSION.

Also is it necessary to wrap the sameSite value in a Some / Option monad? won't `sameSite=Cookie.SameSite.None` do the job?